src/cargo.rs

```diff
@@ -212,17 +211,13 @@ pub fn config_get(key: &str) -> Result<String> {
         anyhow!(
             "Command failed with status: {:#?} and stderr: {:#?}.",
             out.status,
-            OsStr::from_bytes(&out.stderr),
+            String::from_utf8(out.stderr).unwrap(),
         )
     })?;
 
-    let out_str = OsStr::from_bytes(&out.stdout);
+    let out_str = String::from_utf8(out.stdout).unwrap();
```

Comment:
```suggestion
    let out_str = String::from_utf8(out.stdout)?;
```

Should be preferable.